src/model/enums/category-type.ts

```diff
@@ -0,0 +1,5 @@
+export enum CategoryType {
```

Comment:
Please explain why we need Category Type.

Reply:
Referring to the documentation - it is predefined string that should be stored as enum.
It refers to ingredients, articles or recipes.
![image](https://user-images.githubusercontent.com/48626088/113477512-800e1d00-9482-11eb-8bb7-d23a86eabcea.png)

Reply:
We discussed the topic on Slack and agreed that I would remove the type and add a parent category to the model. Done.